src/blocks.js

```diff
@@ -101,8 +54,31 @@ const registerBlock = ( block ) => {
 
 	const { name, settings } = block;
 
-	if ( ! supportsCollections() && ! name.includes( 'gallery' ) ) {
-		category = 'coblocks';
+	// See https://github.com/Automattic/jetpack/issues/14598
+	const availableCategories = wp.blocks.getCategories().map( category => category.slug );
```

Comment:
This is accurately taken from the builder, but it should use the imported `getCategories`:

```suggestion
	const availableCategories = getCategories().map( category => category.slug );
```

Using it via `wp.blocks…` was probably an oversight in 12-gh-coblocks-builder